.github/workflows/update-json-rpcs-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: Update JSON-RPC docs
+
+on:
+  push:
+    paths:
+      - 'tools/generate_json_rpc_docs.py'
+      - 'src/*rpc*.cpp'
```

Comment:
Hmm.
- The upside of running it on push (including on the source branch so that it ends up on PRs) is that the PR contains the rendered file which simplifies preview.
- The downside is that the branch is modified at Github, i.e. further pushes must be forced.
- Another rather important downside is that the branch's latest commit will be created by this workflow, effectively hiding all the other Autobuild-related checks.

I'm therefore restricting this to master for now.

Reply:
Shameless plug: https://github.com/dtinth/patch-generator-action

It fails the build if the working directory is not clean, but also prints a diff that can be applied to the branch.

Reply:
@dtinth I think for this use case it's easier to point the author to running your script directly :)

Reply:
@pljones Done:
https://github.com/jamulussoftware/jamulus/runs/5397974064?check_suite_focus=true#step:4:8
https://github.com/jamulussoftware/jamulus/runs/5397990788?check_suite_focus=true#step:4:5